psydac/api/fem.py

```diff
@@ -1409,6 +1493,9 @@ def construct_arguments(self):
         return args
 
     def assemble(self, **kwargs):
+        """
+        This method assemble the right hand side Vector coefficient with the self._func.
```

Comment:
Some details: 
- typos: assemble**s** , right-hand side
- instead of `Vector coefficient`, maybe `StencilVector` ?

Reply:
[f8b9e17](https://github.com/pyccel/psydac/pull/302/commits/f8b9e1737199588c25955b53eac5150f07c2c0e2)